educe/learning/edu_input_format.py

```diff
@@ -28,13 +31,16 @@ def _dump_edu_input_file(docs, f):
             edu_txt = edu.text().replace('\n', ' ')
             # subgroup: sentence identifier, backoff on EDU id
             sent_idx = edu2sent[i]
-            subgroup = ('{}_sent{}'.format(grouping, sent_idx)
-                        if sent_idx is not None
-                        else edu_gid)
+            if sent_idx is None:
+                subgroup = edu_gid
+            elif isinstance(sent_idx, six.string_types):
```

Comment:
In the case of STAC, EDUs are contained within Turns, which make a somewhat more natural subgrouping than sentences (you could also parse into sentences, but turns are hard game artefacts). These turns have a natural string ID because they also have a representation as annotations. 

On the other hand, we don't strictly speaking *have* to know their ID; it's just a bit nicer to line them up.